inginious/frontend/pages/admin/admin.py

```diff
@@ -23,15 +24,34 @@ def POST_AUTH(self, *args, **kwargs):
 
     def show_page(self):
         """Display page"""
-
-        page = int(request.form.get("page")) if request.form.get("page") is not None else 1
-        user_per_page = 10  # TODO probably better to let user define user_per_page
-        all_users = self.user_manager.get_users_info(usernames=None, limit=user_per_page, skip=(page-1)*user_per_page)
-        size_users = self.user_manager.get_users_count()
-        pages = size_users // user_per_page + (size_users % user_per_page > 0) if user_per_page > 0 else 1
+        user_input = request.form
+        if "displayed_selection" in user_input:
+            # Mostly happened when changing page
+            params = json.loads(user_input.get("displayed_selection", ""))
+        else:
+            params={"limit":10,"sort_by":"username","order":1}
```

Comment:
There is a lot of unrelated data mixed into this dict that are exploded afterwards and compilcates reading and comprehension of the code.